internal/pkg/v1/action/catalog_installed_get_test.go

```diff
@@ -0,0 +1,99 @@
+package action_test
+
+import (
+	"context"
+	"fmt"
+	"slices"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	olmv1catalogd "github.com/operator-framework/catalogd/api/v1"
+
+	internalaction "github.com/operator-framework/kubectl-operator/internal/pkg/v1/action"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
+
+var _ = Describe("CatalogInstalledGet", func() {
```

Comment:
is it worth exercising the sort order?

Reply:
I asked the same question and originally thought that it be too much like testing the built-in cmp functionalities, but I guess an additional test case won't hurt